query-engine/connectors/sql-query-connector/src/database/nodejs.rs

```diff
@@ -9,26 +9,26 @@ use quaint::{
 pub enum RuntimePool {
     Rust(Quaint),
 
-    #[cfg(feature = "nodejs-drivers")]
-    NodeJS(nodejs_drivers::Queryable),
+    #[cfg(feature = "client-drivers")]
+    Client(client_drivers::Queryable),
 }
 
 impl RuntimePool {
-    pub fn is_nodejs(&self) -> bool {
+    pub fn is_client_side(&self) -> bool {
         match self {
             Self::Rust(_) => false,
 
-            #[cfg(feature = "nodejs-drivers")]
-            Self::NodeJS(_) => true,
+            #[cfg(feature = "client-drivers")]
+            Self::Client(_) => true,
         }
     }
```

Comment:
Personally, something like `is_js()` would be quicker to understand and easier to remember than `is_client_side()`

Reply:
Do you want me to rename everything else accordingly as well for consistency (i.e. `client-drivers` to `js-drivers`, `Client` to `JS` and so on), or only this method?

Reply:
Yes, please.